scripts/release.sh

```diff
@@ -361,11 +417,24 @@ function parse_flags() {
             ;;
           --release-gcr)
             KO_DOCKER_REPO=$1
-            has_gcr_flag=1
+            has_registry_flag=1
+            ;;
+          --release-acr)
+            AZ_ACR_NAME=$1
+            KO_DOCKER_REPO="${1}.azurecr.io"
+            has_registry_flag=1
+            ;;
+          --release-azblob)
+            RELEASE_AZ_BLOB_URL=$1
+            has_storage_flag=1
             ;;
           --release-gcs)
             RELEASE_GCS_BUCKET=$1
-            has_gcs_flag=1
+            has_storage_flag=1
+            ;;
+          --test-args)
+            # multiple args can be passed as a|b
+            VALIDATION_TEST_ARGS="$(tr '|' ' ' <<<$1)"
```

Comment:
This is counter-intuitive, pass multiple arguments quoted.

Reply:
I tried very hard to do that - however the quotes passed from the command line disappear causing it to fail :(

Reply:
Quotes indeed won't be part of the string, and you just have to assign the value, no need to use `tr`.

Reply:
removed tr